packages/wrangler/src/d1/timeTravel/info.ts

```diff
@@ -0,0 +1,57 @@
+import { printWranglerBanner } from "../..";
+import { withConfig } from "../../config";
+import { logger } from "../../logger";
+import { requireAuth } from "../../user";
+import { Database } from "../options";
+import { getDatabaseByNameOrBinding } from "../utils";
+import {
+	checkIfDatabaseIsExperimental,
+	getBookmarkIdFromTimestamp,
+} from "./utils";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../../yargs-types";
+
+export function InfoOptions(yargs: CommonYargsArgv) {
+	return Database(yargs)
+		.option("timestamp", {
+			describe:
+				"accepts a Unix (seconds from epoch) or RFC3339 timestamp (e.g. 2023-07-13T08:46:42.228Z) to retrieve a bookmark for",
+			type: "string",
+		})
+		.option("json", {
+			describe: "return output as clean JSON",
+			type: "boolean",
+			default: false,
+		});
+}
+
+type HandlerOptions = StrictYargsOptionsToInterface<typeof InfoOptions>;
+
+export const InfoHandler = withConfig<HandlerOptions>(
+	async ({ database, config, json, timestamp }): Promise<void> => {
+		// bookmark
+		const accountId = await requireAuth(config);
+		const db = await getDatabaseByNameOrBinding(config, accountId, database);
+		await checkIfDatabaseIsExperimental(accountId, db.uuid);
+		const result = await getBookmarkIdFromTimestamp(
+			accountId,
+			db.uuid,
+			timestamp
+		);
+		if (json) {
+			logger.log(JSON.stringify(result, null, 2));
+		} else {
+			await printWranglerBanner();
+			logger.log("🚧 Time Traveling...");
```

Comment:
This needs a cooler emoji for time travel! 🌀 or 🪄

Reply:
will fix in a follow-up PR ✅